_episodes/01-intro.md

```diff
@@ -93,7 +139,7 @@ we can search the documentation.
 Type `read csv` into the documentation toolbar.
 MATLAB suggests using `csvread`.
 If we have a closer look at the documentation,
-MATLAB also tells us, which in- and output arguments this function has.
+MATLAB also tells us, which input and output arguments this function has (try also `help csvread` on the MATLAB shell).
```

Comment:
We made this point with `mean` earlier. Let's skip it with `csvread`; instructors can choose to do this if they like